Logger/FileLogger.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.IO;
+
+namespace Logger;
+
+internal class FileLogger : BaseLogger
+{
+    private string? _Path;
+
+    public FileLogger(string? path)
+    {
+        _Path = path;
+    }
+
+    public override void Log(LogLevel logLevel, string message)
+    {
+        StreamWriter writer = File.AppendText(_Path!);
```

Comment:
```suggestion
        StreamWriter writer = File.AppendText(_Path);
```
Avoid using this whenever you can!! If you take my above suggestions for lines 8-13, you won't need this here anymore.

If you ever find yourself typing `!` ask yourself why you're doing that, and if there is a better solution.

Reply:
Modifying to your above suggestions has indeed removed the need for this operator.